### ISSUE TITLE
[8pt][E4] As the creator of the dashboard/funnel for the OS Climate community, I need the blueprint of the path, so I know the the goal and path a user can & should follow, defined by the chair/host of OS Climate

### ISSUE THREAD

**Issue by hemajv** [open-services-group/community#204]
Once we have an idea of the user journey map operate-first/support#855 for OS Climate, we can start working on implementing a dashboard to visualize the "actions" or "events" that lead to a successful contribution to the community. For eg, Of all the users that created an on-boarding request to the project, how many successfully logged in to the cluster, created an issue, created a PR, reviewed at PR etc.

**Acceptance Criteria**

**Part 1 - Data Acquisition (Sprint May 17-May 31)**

- [x] Identify the required data sources (i.e GitHub, Prometheus etc)
- [x] Store and collect the necessary datasets (i.e S3, Trino)
- [x] Create Trino tables and populate it with relevant datasets

**Part 2 - Data Visualization (Sprint May 31-June 14)**

- [ ] Identify 3-5 metrics to be visualized for tracking community engagement
- [ ] Implement charts in Superset to visualize the metrics
- [ ] Aggregate the charts into a public facing dashboard with descriptive markdowns
- [ ] Share the dashboard and gather feedback from project chair/host/product owner/stakeholder
- [ ] Finalize the dashboard and export it as JSON and store in the project repo (version control)

**Comment by hemajv** [comment]
@aakankshaduggal and I were able to identify required data sources, we will primarily be looking at GitHub data to fetch relevant user journey events.
We have created an initial [notebook](https://github.com/hemajv/metrics/blob/os-climate-data/notebooks/GitHub_Data_Collection.ipynb) that will fetch all GitHub related data such as issues, PRs, events etc. There is also an [ongoing storage outage](https://github.com/operate-first/support/issues/648) in the smaug cluster due to which Trino is unable. Once its resolved, we will work on creating the required tables in Trino.
